client/js/libs/handlebars/parse.js

```diff
@@ -7,6 +7,9 @@ const findLinks = require("./ircmessageparser/findLinks");
 const findEmoji = require("./ircmessageparser/findEmoji");
 const merge = require("./ircmessageparser/merge");
 
+// FIXME: This is just for testing
+const templates = require("../../../views/chat.tpl"); // eslint-disable-line
```

Comment:
This was temporary to force something that would break on `master`, see if tests pass here. I'll remove before calling this PR ready.